packages/libs/src/api/Account.ts

```diff
@@ -187,6 +190,51 @@ export class Account extends Base {
     }
   }
 
+  async guestSignUp(
+    email: string,
+    metadata: UserMetadata,
+    host = (typeof window !== 'undefined' && window.location.origin) || null,
+    generateRecoveryLink = true
+  ) {
+    const phases = {
+      CREATE_USER_RECORD: 'CREATE_USER_RECORD',
+      HEDGEHOG_SIGNUP: 'HEDGEHOG_SIGNUP',
+      SELECT_STORAGE_NODE: 'SELECT_STORAGE_NODE',
+      ADD_USER: 'ADD_USER',
+      UPLOAD_PROFILE_IMAGES: 'UPLOAD_PROFILE_IMAGES'
+    }
+    let phase = ''
+    try {
+      this.REQUIRES(Services.CREATOR_NODE, Services.IDENTITY_SERVICE)
+      this.REQUIRES(Services.HEDGEHOG)
+      phase = phases.HEDGEHOG_SIGNUP
+
+      // guest account secured by OTP
+      const ownerWallet = await this.hedgehog.signUp({
+        username: email,
+        password: 'TemporaryPassword'
```

Comment:
is it unsafe to have this hard coded in the repo?

if I know someone did guest checkout (e.g. by scanning blocks or whatever) can I find and claim temp accounts with this password?

Reply:
This is behind a feature flag and even then the account is secured by OTP